intro.tex

```diff
@@ -0,0 +1,12 @@
+\section{Introduction} \label{sec:intro}
+
+We pull together information from Jira and refer to other documents as needed to describe DP1 in more detail.
+
+DP1 should be released to the community after the First Look image is released to the press.
+
+
+\subsection{DP1 Scope}
+DP1 comprises data from ComCam on sky.
+There will be PVIs.
+There is no promise of coadds but if there is good data this is a stretch goal.
```

Comment:
"difference images or coadds"

Reply:
Everything beyond PVIs and catalogs of detections in single-visit images is a stretch goal.